RoaringBitmap/src/main/java/org/roaringbitmap/art/Node256.java

```diff
@@ -0,0 +1,127 @@
+package org.roaringbitmap.art;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+
+public class Node256 extends Node {
+
+  Node[] children = new Node[256];
+
+  public Node256(int compressedPrefixSize) {
+    super(NodeType.NODE256, compressedPrefixSize);
+  }
+
+  @Override
+  public int getChildPos(byte k) {
+    int pos = Byte.toUnsignedInt(k);
+    if (children[pos] != null) {
+      return pos;
+    }
+    return ILLEGAL_IDX;
+  }
+
+  @Override
+  public Node getChild(int pos) {
+    return children[pos];
+  }
+
+  @Override
+  public void replaceNode(int pos, Node freshOne) {
+    children[pos] = freshOne;
+  }
+
+  @Override
+  public int getMinPos() {
+    for (int i = 0; i < 256; i++) {
+      if (children[i] != null) {
+        return i;
+      }
+    }
+    return ILLEGAL_IDX;
+  }
+
+  @Override
+  public int getNextLargerPos(int pos) {
+    if (pos == ILLEGAL_IDX) {
+      pos = -1;
+    }
+    for (pos++; pos < 256; pos++) {
```

Comment:
A 256 bit bitmask would make this rank search more efficient with small memory overhead.